index.html

```diff
@@ -197,19 +224,20 @@ <h3 class="project-card-title">Multi-Post Stories</h3>
               <li class="technology">bootstrap</li>
               <li class="technology">ruby</li>
             </ul>
-            <button type="button" class="button">See Project</button>     
+            <button type="button" class="button">See Project</button>
+            </section>
           </article>
         </section>
       </section>
       <!-- Contact me section -->
       <section class="contact-section">
-        <h2 class="contact-title">contact us</h2>
+        <h2 class="contact-title">Contact us</h2>
```

Comment:
- [ ] [REQUIRED] For the desktop version, instead of "Contact US" the form heading should change to "Let's chat!" to match the original design. Please correct that and submit for another review😊

Reply:
I am aware of this one but yet again it is not mandatory at this stage of the project as we don't have to implement any functionality involving javascript.
Thus I didn't feel the need to do it as I had no time.
I will be grateful if you check the requirements of the project, the link is provided in a previous reply above ⬆️